packages/fluentui/accessibility/src/behaviors/Tree/treeAsListboxBehavior.ts

```diff
@@ -0,0 +1,31 @@
+import * as _ from 'lodash';
+
+import { Accessibility } from '../../types';
+import { treeBehavior, TreeBehaviorProps } from './treeBehavior';
+import { treeItemAsOptionBehavior } from './treeItemAsOptionBehavior';
+import { treeTitleAsOptionBehavior } from './treeTitleAsOptionBehavior';
+
+/**
+ * @specification
+ * Adds role='listbox'.
+ * Adds attribute 'aria-multiselectable=true' to 'root' slot if 'selectable' property is true. Does not set the attribute otherwise.
+ */
+export const treeAsListboxBehavior: Accessibility<TreeBehaviorProps> = props => {
+  const behavior = treeBehavior(props);
+  const definition = _.merge(behavior, {
+    attributes: {
+      root: {
+        role: 'listbox',
+      },
+    },
+    childBehaviors: {
+      item: treeItemAsOptionBehavior,
+      title: treeTitleAsOptionBehavior,
+    },
+  });
```

Comment:
Let's just use spread instead of `_.merge`, like it's done in [toolbarItemBehavior](https://github.com/microsoft/fluentui/blob/a4898636dfef66f59f63740ab2646c7497410137/packages/fluentui/accessibility/src/behaviors/Toolbar/toolbarItemBehavior.ts#L18-L22)
```suggestion
  const definition = {...behavior, ...{
    attributes: {
      root: {
        role: 'listbox',
      },
    },
    childBehaviors: {
      item: treeItemAsOptionBehavior,
      title: treeTitleAsOptionBehavior,
    },
  }};
```

Reply:
The suggestion way of merging will provide different result than `_.merge`. 
For example, `behavior.attributes.root.tabIndex` is not merged in to `definition`. (codepen: https://codepen.io/amberpie/pen/JjKGPEv) 

Is there any specific reason spread operator is prefered? If so I could change to something like:
```ts
 attributes: {
      root: {
        ...behavior.attributes.root,
        role: 'listbox' as AriaRole,
      },
    },
```

Reply:
Communicate thru chat, going to change from merge to spread in each level :)

Reply:
Resolved by this [commit](https://github.com/microsoft/fluentui/pull/15147/commits/f6d948ee7fb131877a7d22cb2b19b28ea484bb34)